SofaKernel/framework/sofa/core/visual/DrawToolGL.cpp

```diff
@@ -236,6 +236,13 @@ void DrawToolGL::drawTriangles(const std::vector<Vector3> &points, const Vec<4,f
     resetMaterial(colour);
 }
 
+void DrawToolGL::drawTriangles(const std::vector<Vector3> &points, const std::vector< Vec4f > &colour)
+{
+    std::vector<Vector3> normal;
+    normal.clear();
```

Comment:
This means that if the normal is empty it does nothing ? 
Maybe add some comment to explain the logic the declaration of drawTriangles()